--- lib/librte_bbdev/rte_bbdev_version.map	2019-04-08 23:28:53.000000000 +0800
+++ lib/librte_bbdev/rte_bbdev_version.map.patched	2019-05-30 12:49:13.000000000 +0800
@@ -6,11 +6,11 @@
 	rte_bbdev_callback_unregister;
 	rte_bbdev_close;
 	rte_bbdev_count;
-	rte_bbdev_dequeue_dec_ops;
-	rte_bbdev_dequeue_enc_ops;
+	#rte_bbdev_dequeue_dec_ops;
+	#rte_bbdev_dequeue_enc_ops;
 	rte_bbdev_devices;
-	rte_bbdev_enqueue_dec_ops;
-	rte_bbdev_enqueue_enc_ops;
+	#rte_bbdev_enqueue_dec_ops;
+	#rte_bbdev_enqueue_enc_ops;
 	rte_bbdev_find_next;
 	rte_bbdev_get_named_dev;
 	rte_bbdev_info_get;
